youtube_dl/YoutubeDL.py

```diff
@@ -664,6 +665,8 @@ def extract_info(self, url, download=True, ie_key=None, extra_info={},
             ies = self._ies
 
         for ie in ies:
+            if(ie.ie_key() == "Generic" and no_generic_extractor):
```

Comment:
It's idiomatic not to use parentheses in `if` statement. Also use single quotes consistently.